Source/items.h

```diff
@@ -348,6 +349,20 @@ struct ItemStruct {
 			return false;
 		}
 	}
+
+	UiFlags getTextColor() const

```

Comment:
Why is this implemented in the header instead of in the .cpp? Can you move the implementation and leave only the prototype here?

Reply:
Because it's easier for the compiler to inline (e.g. even without LTO) and this function should be inlined.

Reply:
I find these selective rules really, _really_ bad. I'll leave it at that.